src/dev-mongo.js

```diff
@@ -0,0 +1,41 @@
+require("dotenv").config({ path: "./config/.env" });
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const fs = require('fs');
+const path = require('path');
+const mongoose = require('mongoose')
+
+// This file runs a MongoDB server locally for development
+module.exports.run = async () => {
+  const dbPath = path.join(__dirname, '..', '.mongo');
+
+  if (!fs.existsSync(dbPath)) {
+    fs.mkdirSync(dbPath);
+  }
+
+  // start mongo
+  const mongod = await MongoMemoryServer.create({
```

Comment:
A very small nitpick and it might be this way for a reason. Maybe we change this to mongoD?

Reply:
This should likely be mongoDB 

Wouldn't let that stop a merge though. File a bug on it

Reply:
Yeah I wasn't quite sure, several tutorials seems to have it as mongod so it seemed like some format I wasn't familiar with. I'd be okay renaming it to mongoServer as well, as that feels more descriptive.